### PR TITLE
fix: Service ignores timestamp sent by device

### PR DESCRIPTION
**Pull request by marciofunes**
* **Please check if the PR fulfills these requirements**
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by gilmaicor — COMMENTED** [review]
Corrigir os erros apresentados pelo CodeFactor.

File: connector/mqtt/vernemq/v2k-bridge/app/Utils.js [hunk 1]
```diff
@@ -19,12 +19,37 @@ const generateDojotDeviceDataMessage = (topic, payload) => {
...
+  const metadata = { timestamp: 0 };
+
+  if ("timestamp" in payload) {
+    //metadata = { timestamp: 0 };
+    // If it is a number, just copy it. Probably Unix time.
+    if (typeof payload.timestamp === "number") {
+      if (!Number.isNaN(payload.timestamp)) {
+        metadata.timestamp = payload.timestamp;
+      } else {
+        logging.info("Received an invalid timestamp (NaN)");
```
Comment (luissajunior) [inline connector/mqtt/vernemq/v2k-bridge/app/Utils.js]: Declarar a variável logging utilizando a ferramente de Logger

File: connector/mqtt/vernemq/v2k-bridge/app/Utils.js [hunk 2]
```diff
@@ -19,12 +19,37 @@ const generateDojotDeviceDataMessage = (topic, payload) => {
...
+  let metadata = { timestamp: 0 };
+
+  if ("timestamp".indexOf(payload)) {
+    // metadata = { timestamp: 0 };
+    // If it is a number, just copy it. Probably Unix time.
+    if (typeof payload.timestamp === "number") {
+      if (!Number.isNaN(payload.timestamp)) {
+        metadata.timestamp = payload.timestamp;
+      } else {
+        this.logger.info('Received an invalid timestamp (NaN)');
```
Comment (luissajunior) [inline connector/mqtt/vernemq/v2k-bridge/app/Utils.js:31]: You need import Logger tool to use here

Comment (marciofunes) [reply connector/mqtt/vernemq/v2k-bridge/app/Utils.js:32]: Done, added 'const { Logger } = require('@dojot/microservice-sdk');' following example from vernMQ and pass into CodeFactor.

**Review by luissajunior — APPROVED** [review]

**Review by gilmaicor — APPROVED** [review]

**Review by luissajunior — CHANGES_REQUESTED** [review]
Do not need send to master

**Pull request closed without merging**